content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx

```diff
@@ -322,9 +322,9 @@ datasource db {
 
 More information about the `directUrl` field can be found [here](/reference/api-reference/prisma-schema-reference#fields).
 
-### Data Proxy
+### Prisma Accelerate
 
-The [Data Proxy](/data-platform/classic-projects/data-proxy) is a managed external connection pooler built by Prisma that is integrated in the [Prisma Data Platform](/data-platform) and handles connection pooling for you.
+[Prisma Accelerate](/data-platform/accelerate) is a managed external connection pooler and global cache built by Prisma that is integrated in the [Prisma Data Platform](https://console.prisma.io) and handles connection pooling for you.
```

Comment:
Not sure, but maybe we should link to PDP docs here instead of the console route?

Reply:
That's a fair point, @nikolasburk , @ruheni any suggestions?

Reply:
I'm actually not sure if we should merge this now or later when https://github.com/prisma/docs/pull/5501 has been merged... I think both options are troublesome, just not sure which one is more troublesome. Does anyone have thoughts on that?

Reply:
@nikolasburk ping me after you've merged 5501, I think that approach will be easier. I'll put this task to blocked for now.

Reply:
```suggestion
[Prisma Accelerate](/data-platform/accelerate) is a managed external connection pooler and global cache built by Prisma that is integrated in the [Prisma Data Platform](/data-platform) and handles connection pooling for you.
```